sensor_msgs/Image image
float32 left				# 0 is pinned on left edge, 1 is pinned right
float32 bottom				# 0 is pinned to top edge, 1 is pinned to bottom
geometry_msgs/Pose drone_pose   # Position of the drone's camera in absolute space at time the picture was taken
